snappy_pipeline/workflows/ngs_mapping/__init__.py

```diff
@@ -450,7 +451,17 @@ class ReportGetResultFilesMixin:
     """Mixin that provides ``get_result_files()`` for report generation steps"""
 
     def skip_result_files_for_library(self, library_name: str) -> bool:
-        return False
+        if not getattr(self, "tool_categories", None):
```

Comment:
For my understanding: this is the split between long and short reads for DNA sequencing?

Reply:
This is Manuel's code. I am not sure that it solves all possible cases, but at present this is a way to skip `target_coverage` & `bam-collection` reports. When the reporting tool has `tool_categories` in its class attributes, the code for skipping reports based on extraction types is run. When there is no `tool-categories` class attribute, then the report is never skipped.